.github/workflows/build.yml

```diff
@@ -28,12 +28,27 @@ jobs:
             codesign: true
           - os: macos-latest-xlarge
             codesign: true
+          - os: ubuntu-latest
+            musl: true
```

Comment:
How this can coexist with 

```
          - os: ubuntu-latest
            fatjar: true
```

Does it run two independent ubuntu jobs?

Reply:
both run in the same job
I have put them together now, for clarity 
<img width="214" alt="Screenshot 2023-12-18 at 19 53 27" src="https://github.com/seqeralabs/wave-cli/assets/6575288/b24fe7c0-30c3-4698-8fb5-f2fe266a1c5c">